### DISCUSSION TITLE
Showing y value in Marker

### DISCUSSION THREAD

**Discussion by tchouaffe** [patrykandpatrick/vico#301] [Questions] [answered]
Is there a way to also display the _x_ values within the Marker?
For now only the _y_ values are shown in the marker.

**Comment by patrickmichalik** [comment, marked as answer]
Hello! Yes, this is possible. It sounds like you’re using `MarkerComponent`, so you can pass in a custom `MarkerLabelFormatter` implementation or override the `draw` function.

**Reply by tchouaffe** [reply]
I have used a custom `MarkerLabelFormatter` and it works. Thanks

**Reply by patrickmichalik** [reply]
You’re welcome!
